src/snowflake/snowpark/mock/functions.py

```diff
@@ -451,3 +466,20 @@ def mock_endswith(expr1: ColumnEmulator, expr2: ColumnEmulator):
 @patch("row_number")
 def mock_row_number(window: TableEmulator, row_idx: int):
     return ColumnEmulator(data=[row_idx + 1], sf_type=ColumnType(LongType(), False))
+
+
+@patch("parse_json")
+def mock_parse_json(expr: ColumnEmulator):
```

Comment:
Should we rebase on `https://github.com/snowflakedb/snowpark-python/pull/1100`?

Reply:
I will rebase after you merge your PR, for now I will just copy paste the func I need to avoid introducing extra review overhead

Reply:
done rebasing